test/recipes/30-test_hpke.t

```diff
@@ -15,6 +15,6 @@ use OpenSSL::Test::Utils;
 setup("test_hpke");
 
 plan skip_all => "This test is unsupported in a no-ec build"
-    if disabled("ec");
+    if disabled("ec") || disabled("ecx");
```

Comment:
This should be properly handled in test_hpke.

Reply:
Removed ecx here, please check changes in test_hpke.c.
And I changed the KEMID in OSSL_HPKE_SUITE_DEFAULT to P-256 when ECX disabled(hpke.h). I am confused why the short publen case fails(L1389 in test_hpke.c), do you have any ideas?
I will continue to investigate it